packaging/packager

```diff
@@ -90,32 +96,42 @@ do
     if [[ ! -f $i ]]; then # ignore linux-vdso.so.1
         continue
     fi
+
+    if [[ $INCLUDE_LIBC == false ]]; then
+        matched=$(echo $libc_libs | grep --count $i) || true # prevent the non-zero exit status from terminating the script
+        if [ $matched -gt 0 ]; then
+            continue
+        fi
+    fi
     cp $i $PKG_DIR/lib
     filename=`basename $i`
     if [[ -z "${filename##ld-*}" ]]; then
         PKG_LD=$filename # Use this file as the loader
     fi
 done
 
-bootstrap_script="#!/bin/bash\n
-\n
-set -euo pipefail\n
-\n
-exec \$LAMBDA_TASK_ROOT/lib/$PKG_LD --library-path \$LAMBDA_TASK_ROOT/lib \$LAMBDA_TASK_ROOT/bin/$PKG_BIN_FILENAME \${_HANDLER}\n
-"
+bootstrap_script=$(cat <<EOF
+#!/bin/bash
+set -euo pipefail
+export AWS_EXECUTION_ENV=lambda-cpp
+exec \$LAMBDA_TASK_ROOT/lib/$PKG_LD --library-path \$LAMBDA_TASK_ROOT/lib \$LAMBDA_TASK_ROOT/bin/$PKG_BIN_FILENAME \${_HANDLER}
+EOF
+)
 
-bootstrap_script_no_libc="#!/bin/bash\n
-\n
-set -euo pipefail\n
-\n
-exec \$LAMBDA_TASK_ROOT/lib/$PKG_LD --library-path \$LD_LIBRARY_PATH:\$LAMBDA_TASK_ROOT/lib \$LAMBDA_TASK_ROOT/bin/$PKG_BIN_FILENAME \${_HANDLER}\n
-"
+bootstrap_script_no_libc=$(cat <<EOF
+#!/bin/bash
+set -euo pipefail
+export AWS_EXECUTION_ENV=lambda-cpp
+export LD_LIBRARY_PATH=\$LAMBDA_TASK_ROOT/lib:\$LD_LIBRARY_PATH
```

Comment:
The `--library-path` switch causes the paths defined in LD_LIBRARY_PATH to be ignored. See https://linux.die.net/man/8/ld-linux
I'll try switching it to append rather than prepend.